src/test/java/be/seeseemelk/mockbukkit/MockPlayerListTest.java

```diff
@@ -112,7 +111,7 @@ void addOfflinePlayer()
 	}
 
 	@Test
-	@Disabled("OfflinePlayerMock#setOp isn't implemented")
+//	@Disabled("OfflinePlayerMock#setOp isn't implemented")
```

Comment:
Does this need to be commented out? I don't see why it can't be removed entirely.